### ISSUE TITLE
TypeScript error when trying to add a Gantt instance as a partner of a Scheduler

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#9676]
![image](https://github.com/user-attachments/assets/50899257-9df0-42b6-bb06-1bca35178297)

**Comment by carlos00027** [comment]
Is it possible for Gantt to be a partner with a non-Pro Scheduler?

**Comment by matsbryntse** [comment]
@carlos00027 Yes - for sharing time axis settings. If you want to load the Gantt project into the Scheduler, it needs to be a Pro version.

**Comment by matsbryntse-dev** [comment]
After investigation, this is **expected TypeScript behavior** when using separate full package type declarations (`@bryntum/scheduler` + `@bryntum/gantt`).

**Root cause:** Each full bundle (`scheduler.d.ts`, `gantt.d.ts`) contains its own copy of shared types like `TimelineBase`. TypeScript treats these as nominally different types, even though they're structurally identical.

**Solutions:**

1. **Use thin packages** (recommended): Thin packages share base types via imports, so `TimelineBase` is the same type across packages.

2. **Use a combined bundle**: The Gantt full bundle includes SchedulerPro types, so importing both from `@bryntum/gantt` works with compatible types.

Closing as this is by design. Users combining multiple Bryntum products should use thin packages for proper TypeScript compatibility.
